### DISCUSSION TITLE
68040 support

### DISCUSSION THREAD

**Discussion by elmerucr** [dirkwhoffmann/Moira#23] [Ideas]
Hello Dirk,
I was wondering if 68040 support is to be planned somewehere in the future for Moira. I would like to experiment with floating point code in my E64 system.
Best regards, Elmer

**Comment by dirkwhoffmann** [comment]
Hi Elmer,

68030/40 support is unlikely to come soon. My original plan was to implement 68030 support (MMU) and 68040 support (FPU) afterwards. Unfortunately, I failed to acquire working test hardware for the MMU project. I bought an accelerator card for my A500, but I didn't manage to run the simplest MMU test program on it. The vendor was not interested in my bug reports, so I lost interest in the project. I might give it another try in future, but I need to get working test hardware first.

**Reply by elmerucr** [reply]
Ok, thanks for your message and considering this in the future Dirk!
